Source/Core/ArcGISTiledElevationTerrainProvider.js

```diff
@@ -650,6 +657,7 @@ function requestAvailability(that, level, x, y) {
 
     // Mark whole area as having availability loaded
     that._tilesAvailablityLoaded.addAvailableTileRange(
+      level,
```

Comment:
I just saw this - it looked like a missing parameter - I've got no tests or further context on this one! 😛  Will revert later.

Reply:
😮

Reply:
this makes me question `_tilesAvailablityLoaded` even more...